hbase-client/src/main/java/org/apache/hadoop/hbase/client/BalanceResponse.java

```diff
@@ -0,0 +1,126 @@
+/*
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hadoop.hbase.client;
+
+import org.apache.yetus.audience.InterfaceAudience;
+
+/**
+ * Response returned from a balancer invocation
+ */
+@InterfaceAudience.Public
+public final class BalanceResponse {
+
+  /**
+   * Builds a {@link BalanceResponse} for returning results of a balance invocation to callers
+   */
+  @InterfaceAudience.Public
```

Comment:
Thinking about it, we probably don't want the Builder class here to be Public. Users shouldn't ever be building this `BalanceResponse` on their own, just using the `BalanceResponse`.

Reply:
I created https://issues.apache.org/jira/browse/HBASE-26240 for this. I'll submit a small PR once these existing PRs are merged.